01-make_graph.py

```diff
@@ -74,8 +75,12 @@ def source_location(meta_yaml):
 try:
     for i, name in enumerate(total_names):
         print(i, name)
-        feedstock = gh.repository('conda-forge', name + '-feedstock')
-        meta_yaml = feedstock.contents('recipe/meta.yaml')
+        r = requests.get('https://api.github.com/repos/conda-forge/'
+                         '{}-feedstock/contents/recipe/meta.yaml'.format(name),
+                         auth=(os.environ['USERNAME'], os.environ['PASSWORD']))
+        if r.status_code != 200:
+            raise github3.GitHubError(r)
+        meta_yaml = r.json()['content']
         if meta_yaml:
             text = codecs.decode(b64decode(meta_yaml.content))
```

Comment:
`meta.yaml` is just a string now, so should just be `text = codecs.decode(b64decode(meta_yaml))`

Reply:
thank you!